CHANGELOG.md

```diff
@@ -16,6 +16,63 @@
 
 ---
 
+## v4.0.0
+
+Here is the list of all the breaking changes introduced by v4 of the RUM and Logs SDKs.
+
+### [RUM and Logs] Intake URLs change
+
+We changed URLs of where the Browser SDK data is sent. Please make sure that your [Content Security Policy is up to date](https://docs.datadoghq.com/real_user_monitoring/faq/content_security_policy/?tab=us).
+
+### [RUM and Logs] Drop support for TypeScript below v3.8.2
+
+The Browser SDK is now incompatible with TypeScript v3.8.2. If you are using TypeScript, please make
+sure that the version you are using is greater or equal to v3.8.2.
+
+### [RUM and Logs] Tags syntax sanitization
+
+`version`, `env` and `service` [initialization parameters](https://docs.datadoghq.com/real_user_monitoring/browser/#initialization-parameters) are sent as tags to Datadog. The Browser SDK will now sanitize and print a warning if those values don't meet [tag requirements syntax](https://docs.datadoghq.com/getting_started/tagging/#defining-tags).
+
+### [RUM and Logs] Configuration typings cleanup
+
+Some TypeScript types have been cleaned up to better represent accepted initialization parameters
+for each SDKs.
```

Comment:
that could sound a bit cryptic from a customer point of view.
Do we expect them to change something?
if yes, what?
if no, does it worth to communicate on it?

Reply:
I clarified this section. I think it's worth mentioning it since it is still a breaking change. LMKWDYT

Reply:
Much clearer! I've added a follow up